docker/experimental/Dockerfile

```diff
@@ -10,6 +10,9 @@ ARG tf_cuda_compute_capabilities="7.0,7.5,8.0"
 
 ARG tpuvm=1
 ARG build_cpp_tests=0
```

Comment:
Are you able to build cpp tests inside of this dockerfile? We're running into issues (the missing MKL library problem, see the failures on https://xl-ml-test.appspot.com/dashboard) with the image we currently use for cpp ops test, and I'm wondering if this can be used instead.

Reply:
Yes. However, it will increase the image size if we include the CPP tests. Do you know how large they are? If it's not huge, we can include them in the nightly TPU Node image and resolve that test failure.

Reply:
Created an issue to follow up https://github.com/pytorch/xla/issues/4197